sample-docker-app/package.json

```diff
@@ -11,7 +11,8 @@
     "build": "tsc",
     "start": "ts-node index",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build:image": "npm run build && docker build -t sample-docker-app ."
+    "build:image": "npm run build && docker build -t sample-docker-app .",
+    "build:image:arm64": "docker buildx build --platform linux/arm64 -f Dockerfile.arm64v8 -t sample-docker-app-arm64v8 --load ."
```

Comment:
This will need some diagnostics to tell people why it fails — because `buildx` is an experimental feature that needs to be switched on explicitly, requires other system packages, and is not available on all host systems.

Reply:
It would probably require a link to docker documentation with a short explanation in the tutorial on how to "activate" it and also to watch that feature evolve, if the name of the command changes, the quickstart should be updated too.

It makes me think, have you considered to add a simple README.md with a link to the tutorial ?
Unless people only come to this repository after visiting your website, it could be useful.

Reply:
Yes, very good idea!